t/venom/test_suites/test_getlogintoken/15_testcBcA.yml

```diff
@@ -0,0 +1,28 @@
+testcases:
+- name: customB_default
+  steps:
+  - type: customB
+    assertions:
+    - result.foobarb ShouldContainSubstring world
+
+- name: customB_custom
+  steps:
+  - type: customB
+    myWord: "universe"
+    assertions:
+    - result.foobarb ShouldContainSubstring universe
+
+- name: customA_default
```

Comment:
This is the demonstration that we can use an executor inside a test suite which is calling another executor: customA is calling customB.

For our use cases, it means that we can call `get_login_token` inside another executor (for example: `delete_node`) and  use token provided by `get_login_token` inside `delete_node` executor.

Last thing we need to confirm is the syntax to reuse a variable assigned by an executor, inside a test suite, which is calling that executor (current usage of `get_login_token` executor.

Reply:
Exactly